omod/src/main/resources/config.xml

```diff
@@ -65,5 +65,13 @@
 		IdentifierSource.hbm.xml
 	</mappingFiles>
 	
+	<globalProperty>
+		<property>${project.parent.artifactId}.prefixLocationAttributeType</property>
+		<defaultValue>Location Code</defaultValue>
```

Comment:
Not sure it is appropriate for the module to set a defaultValue for this property.

Reply:
@mks-d can we leave out the default value? At my end I feel we need this!

Reply:
I think we should take this out because the module doesn't provide the location attribute type in question, so we end up introducing a decoupling between this reference to "Location Code" and its creation somewhere else.